src/test/rustdoc/intra-doc/field.rs

```diff
@@ -0,0 +1,4 @@
+// @has field/index.html '//a[@href="https://doc.rust-lang.org/nightly/core/ops/range/struct.Range.html#structfield.start"]' 'start'
```

Comment:
Nitpicking: generally we tend to put such checks below the inner attributes. Not a blocker though.